.github/workflows/build_workflow.yml

```diff
@@ -3,17 +3,27 @@ name: CI/CD Build Workflow
 on:
   push:
     branches: [master]
-    paths-ignore: ["docs/**"]
 
   pull_request:
     branches: [master]
-    paths-ignore: ["docs/**"]
 
 env:
   UVCDAT_ANONYMOUS_LOG: False
 
 jobs:
+  pre_job:
+    runs-on: ubuntu-latest
+    outputs:
+      should_skip: ${{ steps.skip_check.outputs.should_skip }}
+    steps:
+      - id: skip_check
+        uses: fkirc/skip-duplicate-actions@master
+        with:
+          paths_ignore: '["**/README.md", "**/docs/**", "**/examples/**", "**/misc/**"]'
```

Comment:
This line will check if the changes only pertain to `paths_ignore`.
https://github.com/fkirc/skip-duplicate-actions#skip-ignored-paths